superset-frontend/src/dashboard/components/PropertiesModal/PropertiesModal.test.jsx

```diff
@@ -16,10 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import { render } from 'spec/helpers/testing-library';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
```

Comment:
We're not accepting enzyme for new changes due to RTL migration initiatives.

Reply:
Thanks! I've refactored this away in HEAD. Might be I'll raise another PR for a linter rule guarding this :D

Reply:
My mistake in the first place! Indeed, we don't want to migrate RTL tests to Jest/Enzyme tests, so this would be a step backward. For _existing_ J/E tests, we can use `styledMount`, but we ultimately want to move away from Jest/Enzyme entirely (over time).

Reply:
I saw other folk [attempting to bump Jest to v29](https://github.com/apache/superset/pull/26182) and removed this skipped [test](https://github.com/apache/superset/pull/26182/files#diff-1cd4fcb2ea1a28cdc7d46679305346e7b8e6972c73f9c1a27b98cde2ade726b5)

I think it's a sensible approach to solve both Jest uplift and Enzyme-to-RTL problems. Wdyt?

Reply:
@hainenber What is it about this test file that is obstructing the Jest upgrade?
The entire code is being skipped, but since we are planning to fix this code through the TypeScript migration, I would like to keep this test code as it is.

Reply:
This test, unchanged, would fail when running with Jest v29. However, I use `styledMount` and that managed to get the test passed. But `styledMount` is still using Enzyme under the hood, which is not acceptable due to RTL initiative.

I'm not sure what step I can take here. Are you OK with this test upgrading with `styledMount` to pass Jest test? :?

Reply:
Understood. In that case, rather than deleting the file, please rename it to PropertiesModal.testSkipped.jsx